error:- If we attempt to access a data item with a key, which is not part of the dictionary, we get an error as follows âˆ’

dict = {'Name': 'Zara', 'Age': 7, 'Class': 'First'}

print "dict['Alice']: ", dict['Alice']